frontend/src/components/navigation.jsx

```diff
@@ -1,54 +1,58 @@
 import React from "react";
 import { Link, withRouter } from "react-router-dom";
 
+const active = "bg-yellow rounded text-gray";
+
 function Navigation(props) {
   return (
     <div className="navigation">
-      <nav className="navbar navbar-expand navbar-dark bg-dark">
-        <div className="container">
-          <Link className="navbar-brand" to="/">
-            Lockers &amp; Noise
-          </Link>
+      <nav className="navbar navbar-expand navbar-dark border-b border-yellow">
+        <div className="container w-full flex flex-row justify-between">
+          <div class="flex flex-row justify-self-start flex-grow justify-start">
```

Comment:
react complains about 'class' vs 'className'. Is it a problem or something to ignore?